### PR TITLE
Remove unneeded CSS prefixes and length unit symbols

### PR DESCRIPTION
**Pull request by valtlai**
- Remove `-webkit-` prefixes from `transition` and flexbox-related properties
- Remove `-webkit-` prefix inside unprefixed `@keyframes`
- Add missing unprefixed `transform`s
- Remove `px` unit from `0px` values

(+ Remove whitespace from end of lines)

### CODE REVIEW

**Comment by petetnt** [comment]
LGTM

**Comment by redmunds** [comment]
@valtlait 

The original email notification I received said this:

> (+ Remove whitespace from end of lines and empty lines)

But message in PR now says this:

> (+ Remove whitespace from end of lines)

It looks like it was edited to remove "and empty lines". But, I still see lines with only whitespace that were edited to remove whitespace in this PR. Please backout those changes and rebase into a single commit. Thanks.

In general:

Yes, please remove whitespace from the end of non-empty lines.

Please do _not_ remove whitespace from empty lines. This is not part of the Brackets coding conventions. It's optional based on developer tastes. It doesn't do any good to remove it because the next dev might put it back. **Worst of all**, the change will show up as a merge conflict for the next poor soul that edits the file, so it causes unnecessary pain.

Exception: whitespace on empty lines should be at correct indentation level, so it's ok to correct that. But no need to scan files for this -- just correct it in code blocks that you are editing for some other reason.

**Comment by petetnt** [comment]
FWIW There was discussion about this on #11998 too and @peterflynn suggested (for the code) `"no-trailing-spaces": [2, { "skipBlankLines": true }]` which I think makes some sense. The thing is that when `no-trailing-spaces` is enabled you can't really use an automatic whitespace sanitizer, without doing tons of silly diffs all over the documents. But then again CodeMirror likes to put whitespace allover the place so there's you have either to endure the manual removal of trailing spaces, let them be or just remove them all, all of which have their downsides.

**Comment by valtlait** [comment]
Reverted whitespace changes for empty lines.

**Comment by redmunds** [comment]
@valtlait Thank you!

File: src/styles/brackets.less [hunk 1]
```diff
@@ -511,8 +511,7 @@ a, img {
             margin-bottom: 0;
             .sprite-icon(0, 0, 13px, 13px, "images/flip-view-icons.svg");
             background-origin: content-box;
-            -webkit-transform: translateZ(0); // forces GPU mode for better filter rendering on retina
-            transform: translateZ(0); // future proofing
+            transform: translateZ(0); // forces GPU mode for better filter rendering on retina
```
Comment (MarcelGerber) [inline src/styles/brackets.less]: Please note that the Linux build of Brackets Shell is sadly still based on CEF 1547 (~ Chrome 29), so you still need these prefixes (and probably others, too) according to caniuse.com

**Comment by valtlait** [comment]
* Restored `-webkit-` prefix to `transform`
* Removed `-webkit-` prefix inside unprefixed `@keyframes`
* Added missing unprefixed `transform`s

File: src/styles/brackets.less [hunk 2]
```diff
@@ -220,8 +220,8 @@ a, img {
 
     #status-language {
         border-right: 1px solid @bc-panel-border;
-        padding: 0px;
```
Comment (MiguelCastillo) [inline src/styles/brackets.less]: Curious. Why did you removed the units from all 0s?

Comment (valtlait) [inline src/styles/brackets.less]: There’s a practice to leave off the unit from zero length values. It makes the code look slightly cleaner.

**Comment by ficristo** [comment]
@valtlai could you update the PR?
It LGTM but I prefer if someone else could do the last check.

**Comment by valtlai** [comment]
@ficristo Updated.

**Comment by ficristo** [comment]
@valtlai the merge didn't end well, can you check?

**Comment by valtlai** [comment]
Replaced with #12648.

**Pull request closed without merging**